api/src/config.rs

```diff
@@ -250,6 +250,7 @@ pub struct BackendConfigV2 {
     /// Configuration for container registry backend.
     pub registry: Option<RegistryConfig>,
     /// Configuration for local http proxy.
+    #[serde(rename = "http-proxy")]
     pub http_proxy: Option<HttpProxyConfig>,
```

Comment:
This deserves a dedicated PR:)

Reply:
Here's the pr https://github.com/dragonflyoss/image-service/pull/1086 .